integration_test/cloudbuild-integration-test.yaml

```diff
@@ -420,11 +517,14 @@ substitutions:
   _LOCATION: us-central1
   _DATASET: healthcare-dicom-dicomweb-adapter-test
   _STORE_NAME: integration-test-store
+  _BUCKET: dicomweb-import-adapter-integration-test
```

Comment:
I have not created this bucket yet, so the integration test shouldn't pass, but it does. Are we not propagating back errors to the store SCU user when we fail to write to a GCS bucket? 

Also once we find out why this is passing, I think we should store different example files for each adapter, and add diff steps for the extra files. Because if the input file is a duplicate the server doesn't do anything.
Meaning that we should have example2.dcm and example3.dcm in steps run-store-scu-with-local-backup and run-store-scu-with-gcs-backup and then extra check-store-curl and check-store-diff commands for each of them. This will also affect the step numbers at the bottom of the file

Reply:
We tested the GCS backup on our project and everything went well. Perhaps the error is encapsulated in the docker container and not translated out? Is it possible to check the container logs?

Reply:
Yeah I guess the store scu command doesn't return a non-zero error code on storage failure so the test won't fail. This highlights the need for storing separate files.